lib/aws/session_store/dynamo_db/configuration.rb

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 
 require 'yaml'
-require 'aws-sdk-v1'
+require 'aws-sdk'
 
 module AWS::SessionStore::DynamoDB
```

Comment:
@awood45 it would be nice to change the top-level module, but I'd do that in it's own PR to make code review simpler which is why I didn't bother for now.